lib/jasmine-core/jasmine.js

```diff
@@ -144,6 +144,7 @@ getJasmineRequireObj().requireMatchers = function(jRequire, j$) {
       'toBeUndefined',
       'toContain',
       'toEqual',
+      'toHaveSize',
```

Comment:
Please remove the generated `jasmine.js` file [as mentioned in the contributing guide](https://github.com/jasmine/jasmine/blob/master/.github/CONTRIBUTING.md)  from the PR so we don't have to deal manage a number of conflicts in this file.

Reply:
ok

Reply:
see 0dfdd2d8a896929b690a2eaa47a7a8369b56d6f6